doc/nrf-bm/release_notes/release_notes_0.8.0.rst

```diff
@@ -0,0 +1,257 @@
+    * Incorrectly stated maximum ATT MTU value.
+
+..
+
+* ``bm_buttons`` library:
+
+  * Added the use of nrfx critical section API.
+
+Samples
+=======
+
+* General changes:
+
+  * All samples now use the logging module instead of ``printk``.
+
+  * Updated:
+
+    * The main loop sleep instructions as the :c:func:`sd_app_evt_wait` function is deprecated.
+    * Updated the main function flow to always end up in a loop with sleep instructions and deferred log processing instead of returning from the main function.
+      This ensures that log messages are printed and that the device goes into sleep mode.
+      Returning from main leaves the device in a state that prevents low power sleep.
+      Unhandled deferred log messages are not processed.
+
+Bluetooth samples
+-----------------
+
+* Added the :ref:`ble_pwr_profiling_sample` sample.
+
+..
+
+* :ref:`ble_cgms_sample` sample:
+
+  * Fixed an issue with responding twice to a PHY update request.
+
+..
+
+* :ref:`ble_hrs_sample` sample:
+
+  * Removed the unused Coded PHY Kconfig fragment.
+
+..
+
+* :ref:`ble_nus_sample` sample:
+
+  * Added support for Low Power UARTE (LPUARTE).
+
+..
+
+  * Fixed:
+
+    * The pin configuration to use the pins from the :file:``board_config.h`` board header.
+    * An issue in the :c:func:`ble_nus_data_send` function where passing NULL as the NUS instance could cause a segmentation fault.
+
+Peripheral samples
+------------------
+
+* Added the following peripheral samples:
+
+  * :ref:`bm_zms_sample` sample.
+  * :ref:`bm_lpuarte_sample` sample.
+  * :ref:`storage_sample` sample.
+
+DFU samples
+-----------
+
+* Added the following samples:
+
+  * :ref:`ble_mcuboot_recovery_entry_sample` sample that demonstrates how to activate DFU mode on the device using a mobile application and execute DFU over Bluetooth LE.
+  * :ref:`ble_mcuboot_recovery_retention_sample` sample that demonstrates how to switch a device into DFU mode on embedded application's request and execute DFU over UART or Bluetooth LE.
+
+Known issues and limitations
+============================
+
+* Some issues are observed when using iPhone as the peer during testing of the Bluetooth samples.
+* Some issues are observed when using Linux with the Bluetooth Low Energy app in nRF Connect for Desktop as the peer during testing of the Bluetooth samples.
+* There are no samples or libraries available to demonstrate or support the Bluetooth LE Pairing and bonding functionality in the SoftDevice.
+* The samples are not optimized for power consumption unless explicitly stated.
+
+Documentation
+=============
+
+* First public release of product documentation.
+
+..
+
+* Added:
+
+  * Top-level sections for :ref:`drivers`, :ref:`libraries`, :ref:`bm_dfu`, and :ref:`s115_docs`.
+  * Documentation for the :ref:`lib_ble_conn_params` library.
+  * :ref:`nrf_bm_api` section.
+
+* :ref:`s115_docs` are now integrated into |BMshort| documentation rather than delivered as PDF files.
```

Comment:
PDF files are still in bm repo, should we remove them going forward?

Reply:
We don't have to, some customers might prefer PDF. I will remove the 2nd part of the sentence.
*  :ref:`s115_docs` are now integrated into the |BMshort| documentation.